docs/en/concept/sql-config.md

```diff
@@ -0,0 +1,187 @@
+# SQL Configuration File
+
+## Structure of SQL Configuration File
+
+The `SQL` configuration file appears as follows.
+
+### SQL
+
+```sql
+/* config
+env {
+  parallelism = 1
+  job.mode = "BATCH"
+}
+*/
+
+CREATE TABLE source_table WITH (
+  'connector'='jdbc',
```

Comment:
Is it better for this field value refer to [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties)?